### DISCUSSION TITLE
모든 제페토 월드에서 쓸 수 있는 아바타(모자 등)는 게임에서 파는 게 가능할까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#877] [5. Q&A]
관련한 문서가 없이
소모성, 비소모성 아이템 관한 내용만 보여서 확인 드립니다:)

판매할 아바타 등록하는 문서는 있는데, 그걸 게임에 불러오는 게 안보여요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 현재는 크리에이터 코스튬 아이템은 제페토 앱 내부 상점에서만 구입할 수 있습니다.
코스튬 아이템을 월드 내에서 판매가 가능하게 하기 위해서는 연동 작업부터 정책까지 고려할 내용이 많아서
지원 준비 중에 있으나 시간이 많이 소요되는 점 양해 부탁 드립니다.
감사합니다.
